### PR TITLE
chore: modernize f-string usage

### PR DESCRIPTION
**Pull request by prusnak**

### CODE REVIEW

File: cashu/wallet/wallet.py [hunk 1]
```diff
@@ -349,9 +348,8 @@ async def load_proofs(self, reload: bool = False, all_keysets=False) -> None:
                 for keyset_id in self.keysets:
                     proofs = await get_proofs(db=self.db, id=keyset_id, conn=conn)
                     self.proofs.extend(proofs)
-        logger.trace(
-            f"Proofs loaded for keysets: {' '.join([k.id + f' ({k.unit})' for k in self.keysets.values()])}"
-        )
+        proofs_str = ' '.join([f"{k.id} ({k.unit})" for k in self.keysets.values()])
+        logger.trace(f"Proofs loaded for keysets: {proofs_str}")
```
Comment (callebtc) [inline cashu/wallet/wallet.py:352]:
```suggestion
        keysets_str = ' '.join([f"{k.id} ({k.unit})" for k in self.keysets.values()])
        logger.trace(f"Proofs loaded for keysets: {keysets_str}")

```

File: cashu/nostr/bech32.py [hunk 2]
```diff
@@ -73,7 +73,7 @@ def bech32_create_checksum(hrp, data, spec):
 def bech32_encode(hrp, data, spec):
     """Compute a Bech32 string given HRP and data values."""
     combined = data + bech32_create_checksum(hrp, data, spec)
-    return hrp + "1" + "".join([CHARSET[d] for d in combined])
+    return f"{hrp}1{''.join([CHARSET[d] for d in combined])}"
```
Comment (callebtc) [inline cashu/nostr/bech32.py:76]: In some places, the f-string format is a bit less intuitive

**Comment by callebtc** [comment]
@prusnak thanks! it's hard to see what changed with the force pushes. I like this PR in almost all places but some strings are definitely harder to read as f-strings. Did you consider the comments above or are they "resolved" because of the force-push?

**Comment by prusnak** [comment]
> Did you consider the comments above or are they "resolved" because of the force-push?

Yes, I addressed both comments.

**Pull request merged**